packages/foreman/foreman/foreman.spec

```
@@ -706,6 +708,7 @@ plugins required for Foreman to work.
   done
   # script content
   sed -ri 'sX/usr/bin/rakeX%{scl_rake}X' extras/dbmigrate script/foreman-rake
+  sed -i '$ i\source scl_source enable tfm' script/foreman-puma-status
```

I have no idea what this `sed` expression does.

Is it inserting `source scl …` before the ` puma-status /run/foreman/puma.state` in the script?
Wouldn't it be easier to read if it'd prepend that line with a `scl enable tfm -- `?

---

I think inserting the line at line 2 is more straight forward.
```suggestion
  sed -i '2isource scl_source enable tfm' script/foreman-puma-status
```

I think adding a source line is more future proof. If the script is ever changed to multiple lines, it'll still work. You also know that the first line will remain the shebang so line 2 should be safe.